.github/workflows/test.yml

```diff
@@ -0,0 +1,45 @@
+name: Run tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions: {}
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      LOGS_BUCKET_NAME: moj-analytics-s3-logs
```

Comment:
nit: can you move this below permissions?

Reply:
Also actually, what is this for? where is it used?

Reply:
adjusted permissions - the bucket is required for the tests to run - I did a run removing this bucket to test this